### ISSUE TITLE
F-Droid fails to build

### ISSUE THREAD

**Issue by licaon-kter** [yuttadhammo/BodhiTimer#29]
Ref: https://f-droid.org/wiki/index.php?title=org.yuttadhammo.BodhiTimer/lastbuild_70&oldid=286037

Ref: https://gitlab.com/fdroid/fdroidserver/-/blob/2.0b1/fdroidserver/scanner.py#L141

Ref: https://gitlab.com/fdroid/fdroidserver/-/issues/858

**Comment by tzugen** [comment]
Should be fixed in https://github.com/yuttadhammo/BodhiTimer/commit/dd3d52a609ab94233dd8d9f352d8c6626e6a10ba

**Comment by licaon-kter** [comment]
```
2021-03-19 14:03:00,541 DEBUG: buildserver >
2021-03-19 14:03:00,542 DEBUG: buildserver > > Task :app:compileReleaseKotlin
2021-03-19 14:03:00,542 DEBUG: buildserver > w: /home/vagrant/build/org.yuttadhammo.BodhiTimer/app/src/main/java/org/yuttadhammo/BodhiTimer/Models/AlarmTaskManager.kt: (149, 13): Name shadowed: offset
2021-03-19 14:03:00,542 DEBUG: buildserver > w: /home/vagrant/build/org.yuttadhammo.BodhiTimer/app/src/main/java/org/yuttadhammo/BodhiTimer/Models/AlarmTaskManager.kt: (361, 9): Visibility modifiers are redundant in getter
2021-03-19 14:03:00,542 DEBUG: buildserver > w: /home/vagrant/build/org.yuttadhammo.BodhiTimer/app/src/main/java/org/yuttadhammo/BodhiTimer/Util/Sounds.kt: (83, 17): Condition 'mUri != null' is always 'true'
2021-03-19 14:03:00,542 DEBUG: buildserver >
2021-03-19 14:03:00,542 DEBUG: buildserver > > Task :app:compileReleaseJavaWithJavac
2021-03-19 14:03:00,543 DEBUG: buildserver > warning: /home/vagrant/android-sdk/build-tools/30.0.1/core-lambda-stubs.jar(java/lang/invoke/LambdaMetafactory.class): major version 53 is newer than 52, the highest major version supported by this compiler.
2021-03-19 14:03:00,543 DEBUG: buildserver >   It is recommended that the compiler be upgraded.
2021-03-19 14:03:00,543 DEBUG: buildserver > Note: Some input files use or override a deprecated API.
2021-03-19 14:03:00,543 DEBUG: buildserver > Note: Recompile with -Xlint:deprecation for details.
2021-03-19 14:03:00,543 DEBUG: buildserver > 1 warning
2021-03-19 14:03:00,543 DEBUG: buildserver >
2021-03-19 14:03:00,544 DEBUG: buildserver > > Task :app:compileReleaseSources
2021-03-19 14:03:00,544 DEBUG: buildserver > > Task :app:mergeReleaseNativeDebugMetadata NO-SOURCE
2021-03-19 14:03:00,544 DEBUG: buildserver >
2021-03-19 14:03:00,544 DEBUG: buildserver > > Task :app:lintVitalRelease
2021-03-19 14:03:00,544 DEBUG: buildserver > /home/vagrant/build/org.yuttadhammo.BodhiTimer/app/src/main/res/values-de/strings.xml:93: Error: "disable_battery_optim" is translated here but not found in default locale [ExtraTranslation]
2021-03-19 14:03:00,544 DEBUG: buildserver >     <string name="disable_battery_optim">Disable battery optimization</string>
2021-03-19 14:03:00,545 DEBUG: buildserver >             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
2021-03-19 14:03:00,545 DEBUG: buildserver > /home/vagrant/build/org.yuttadhammo.BodhiTimer/app/src/main/res/values-de/strings.xml:94: Error: "disable_battery_optim_desc" is translated here but not found in default locale [ExtraTranslation]
2021-03-19 14:03:00,545 DEBUG: buildserver >     <string name="disable_battery_optim_desc">BodhiTimer needs to be excluded from battery optimization in order for the alarm to work precisely</string>
2021-03-19 14:03:00,545 DEBUG: buildserver >             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
2021-03-19 14:03:00,545 DEBUG: buildserver >
2021-03-19 14:03:00,545 DEBUG: buildserver >    Explanation for issues of type "ExtraTranslation":
2021-03-19 14:03:00,545 DEBUG: buildserver >    If a string appears in a specific language translation file, but there is
2021-03-19 14:03:00,546 DEBUG: buildserver >    no corresponding string in the default locale, then this string is probably
2021-03-19 14:03:00,546 DEBUG: buildserver >    unused. (It's technically possible that your application is only intended
2021-03-19 14:03:00,546 DEBUG: buildserver >    to run in a specific locale, but it's still a good idea to provide a
2021-03-19 14:03:00,546 DEBUG: buildserver >    fallback.)
2021-03-19 14:03:00,546 DEBUG: buildserver >
2021-03-19 14:03:00,546 DEBUG: buildserver >    Note that these strings can lead to crashes if the string is looked up on
2021-03-19 14:03:00,547 DEBUG: buildserver >    any locale not providing a translation, so it's important to clean them
2021-03-19 14:03:00,547 DEBUG: buildserver >    up.
2021-03-19 14:03:00,547 DEBUG: buildserver >
2021-03-19 14:03:00,547 DEBUG: buildserver > 2 errors, 0 warnings
```

**Comment by tzugen** [comment]
Thanks a lot for reporting :)

, should be fixed with 5.6.2
